### ISSUE TITLE
Unknown Error on WOSAC Validation Server Submission

### ISSUE THREAD

**Issue by jxmmy7777** [waymo-research/waymo-open-dataset#882]
Hi,
We encountered an unknown error while submitting the validation_set to the validation server. We followed the instructions in [this tutoria](https://github.com/waymo-research/waymo-open-dataset/blob/master/tutorial/tutorial_sim_agents.ipynb)l to generate the submission files.
Details:

> Unknown error, please file a new issue at https://github.com/waymo-research/waymo-open-dataset/issues/new and specify submission_key=user_id: "weijer_chang@berkeley.edu" submission_id: "1731052631269000".

Thank you for your assistance

**Comment by Dylan-LDR** [comment]
Same issue. 
![image](https://github.com/user-attachments/assets/4938e8a9-796f-41fc-b03f-c6b075b6b94c)
How can we find the bug?

**Comment by zywangEcho** [comment]
Hi! I meet the same error as you, do you solve this problem?

**Comment by jxmmy7777** [comment]
@zywangEcho  yes, my case is that I didn't store 32 joint samples but only store 3, I suggest you to validate the submission before submission `submission_specs.validate_joint_scene(joint_scene, scenario)` in [link](https://github.com/waymo-research/waymo-open-dataset/blob/master/tutorial/tutorial_sim_agents.ipynb)
